1.10-graphql-own/resolvers/post.resolvers.js

```diff
@@ -0,0 +1,46 @@
+const { UserInputError } = require("apollo-server");
+const axios = require("axios");
+
+module.exports = {
+  resolvers: {
+    Query: {
+      posts: async () => {
+        const { data } = await axios.get("http://localhost:3001/post");
+        return data;
+        // return data.map(async (post) => {
+        //   console.log(post);
+        //   const result = await axios.get(
+        //     `http://localhost:3001/authors?id=${post.authorId}`
+        //   );
+        //   return {
+        //     ...post,
+        //     author: result.data[0],
+        //   };
+        // });
+      },
+    },
+    Mutation: {
+      newPost: async (_, { title, author }) => {
+        const { data: postAuthor } = await axios.get(
+          `http://localhost:3001/author?id=${author}`
```

Comment:
I'd recommend defining a `baseUrl` variable to store the URL and port to interpolate it on each query as follows:
```js
await axios.get( `${baseUrl}?id=${author}`);
```